### ISSUE TITLE
Checkbox Accessibility Issue

### ISSUE THREAD

**Issue by MI6-55** [mi6/ic-ui-kit#1886]
## Summary of the bug 
For the Checkbox component the contrast of the grey to a white background is not strong enough (#A7ACB4) 2.28:1. It also seems to be a different grey from all other components with a grey border 

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to page: [Checkbox ](https://design.sis.gov.uk/components/checkbox)
2. Click on: Check border colour of component in non-focused state #A7ACB4
3. Type in: Check contrast against white background and get 2.28:1
4. This does not meet [WCAG criteria 1.4.11 ](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

## 📸 Screenshots or code 
![image](https://github.com/mi6/ic-ui-kit/assets/121944166/c8700c93-3e77-4d51-a0d4-692108896e55)

## 🖥 📱 Device 
- Type: Desktop
- Browser: Chrome

## 🧐 Expected behaviour 
The contrast should be at least 3:1, this could be the same colour as other interactive elements like the text field with #8A919B

## 📝 Acceptance Criteria  
If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil. 

The colour contrast should be at least 3:1.

## Additional info 
Tell us anything else useful to help us fix or understand the problem.
Test with automated or manual tools to ensure contrast is sufficient.

**Comment by MI6-255** [comment]
Worth making sure design have this change
